README.md

```diff
@@ -87,6 +87,7 @@
   - [USE Encoder](jaseci_ai_kit/jaseci_ai_kit/modules/use_enc/README.md)
   - [USE QA](jaseci_ai_kit/jaseci_ai_kit/modules/use_qa/README.md)
   - [Personalized Head](jaseci_ai_kit/jaseci_ai_kit/modules/ph/README.md)
+  - [Whisper](jaseci_ai_kit/jaseci_ai_kit/modules/whisper/README.md)
```

Comment:
Can you also create a new entry in the readme at the top of jaseci_ai_kit? You can create a new table for speech related modules.

Reply:
Can you also make one for the translation module too? I just merged that one.